projects/canopy/src/lib/link-menu/link-menu-item/link-menu-item.component.html

```diff
@@ -4,5 +4,5 @@
 </div>
 <div class="lg-link-menu-item__icon-container">
   <lg-icon [name]="internal ? 'chevron-right' : 'link-external'" class="lg-font-size-0-8"></lg-icon>
```

Comment:
Could the `internal` input be removed and use the `openInANewTab` instead here? 

Also, if you do that, could you please remove the `internal` reference in the docs?

Reply:
I've made these improvements and have updated the example in the docs so the last link-menu-item has the `target="_blank"` attribute set:

![image](https://user-images.githubusercontent.com/8008207/203589016-75ebdfda-d533-4d38-b02b-b4f2d8ae0a1e.png)